contrib/inventory/foreman.py

```diff
@@ -198,6 +266,52 @@ def _get_json(self, url, ignore_errors=None, params=None):
                 break
         return results
 
+    def _use_inventory_report(self):
+        if not(self.foreman_use_reports_api):
```

Comment:
The braces aren't needed
```suggestion
        if not self.foreman_use_reports_api:
```